pkg/gitreceive/build.go

```diff
@@ -201,13 +204,12 @@ func build(conf *Config, s3Client *s3.S3, kubeClient *client.Client, builderKey,
 	if err != nil {
 		return fmt.Errorf("Timed out waiting for object in storage. Aborting build...")
 	}
-
 	log.Info("Build complete.")
 	log.Info("Launching app.")
 	log.Info("Launching...")
 
 	buildHook := &pkg.BuildHook{
-		Sha:         gitSha.Full(),
+		Sha:         gitSha.Short(),
```

Comment:
@smothiki related to my general comment on this PR. This changes https://github.com/deis/builder/pull/131/files#diff-91e10a0836e513c85626cee1d2899e5dR236 back to short. What has changed in controller since that PR 4 days ago?